examples/restaurant_demo/README.md

```diff
@@ -0,0 +1,260 @@
+# Example using the Differential Privacy Library PyDP
+
+## Restaurant
+
```

Comment:
pls add google's dp java example link as ref. 
You may also look at the intro from here as a reference #311

Reply:
I have added Google Java as well as our notebook link for reference